hearth/lib/hearth/retry/retry_quota.rb

```diff
@@ -19,13 +19,14 @@ def initialize
       # Check if there is sufficient capacity to retry and return it.
       # If there is insufficient capacity, return 0
       # @return [Integer] The amount of capacity checked out
-      def checkout_capacity(error_inspector)
+      def checkout_capacity(error_type)
```

Comment:
Design doc says to take a response here, but we can't do that anymore if this is a detail of RetryStrategy. It could take error_inspector again if we do away with error_info.